### PR TITLE
DR-Add Dkim information to app

### PR DESCRIPTION
**Pull request by mvillaseco**
Changes:
 - Add final Content.
 - Add new controller.
 - Add new view.
 - Add link in domains table
 - Add test to be improved.

### CODE REVIEW

File: src/spec/page-objects/settings-page.js [hunk 1]
```diff
@@ -81,5 +85,19 @@ class SettingsPage {
...
   }
+  clickFirstDkimInformationButton(){
+    return this._dKimInformationButtons.then(function(val){
+      return val[0].click();
+    });
+  }
+  getdKimDomainSelected(){
+    return this._dKimDomainSelected.getText();
+  }
+  getdKimDomainSelector(){
```
Comment (andresmoschini) [inline src/spec/page-objects/settings-page.js]: These two fields seems to be part of another page, I am not sure if `settings-page` is a good place for them.

File: src/spec/settings_spec.js [hunk 2]
```diff
@@ -200,4 +200,37 @@ describe('Settings Page', () => {
...
+      .run($httpBackend => {
+        $httpBackend.whenGET(/\/accounts\/[\w|-]*\/domains/).respond(200, {
+          "domains": [{name: "relay.com", dkim_selector: "test", dkim_public_key: "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQC"}, {name: "fromdoppler.com" }, {name: "makingsense.com", disabled: true }, {name: "makingsense12.com" }],
+          "default": "relay.com"
+        });
+      }));
+    var domain = 'relay.com';
+    var dkimSelector = 'test';
+    var dkimPublicKey = 'MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQC';
+    var settings = new SettingsPage();
```
Comment (andresmoschini) [inline src/spec/settings_spec.js]: It is easy to understand the name `settings` in the wrong way, what do you think about `settingsPage` or simply `page`?

File: src/spec/settings_spec.js [hunk 3]
```diff
@@ -200,4 +200,37 @@ describe('Settings Page', () => {
...
+    browser.get('/#/settings/domain-manager');
+
+    //Act
+    settings.clickFirstDkimInformationButton().then(function(){
+      browser.getAllWindowHandles().then(function (handles) {
```
Comment (andresmoschini) [inline src/spec/settings_spec.js]: Maybe all of this code could be moved inside `settings-page`, something like:

```js
settings.clickFirstDkimInformationButton().then(() =>{
    settings.switchToNewTab().then(dkimDetailsPage =>{
        expect(dkimDetailsPage.getdKimDomainSelected()).toBe(domain);
        expect(dkimDetailsPage.getdKimDomainSelector()).toBe(dkimSelector);
    });
});
```

**Comment by andresmoschini** [comment]
Lovely!

**Comment by andresmoschini** [comment]
Great!

File: src/wwwroot/locales/en-translation.js [hunk 4]
```diff
@@ -220,5 +220,9 @@
   "domains_text": "Domain Manager",
   "enable_text": "Enabled",
   "domain_manager_error": "It seems that someone else is using the account at this moment and it not possible to perform the action. Please click the button below to reload the information.",
-  "domain_manager_error_button": "Reload your data"
+  "domain_manager_error_button": "Reload your data",
+  "dkim_configuration": "DKIM Configuration",
+  "dkim_configuration_help_domain_title": "Find here all the necessary information to configure the DKIM for the following domain",
```
Comment (nreal) [inline src/wwwroot/locales/en-translation.js]: This line lack `:` as in Spanish

**Pull request merged**